lib/blpapi-wrapper.ts

```diff
@@ -334,7 +348,25 @@ export class Session extends events.EventEmitter {
         });
     }
 
-    subscribe(subscriptions: Subscription[], cb?: (err: any) => void): Promise<void> {
+    request(uri: string,
+            requestName: string,
+            request: any,
+            identity: blpapi.Identity,
+            callback: IRequestCallback): void
+    {
+        this.doRequest(uri, requestName, request, callback, false, identity);
+    }
+
+    authorizeUser(request: any, callback: IRequestCallback): void {
+        var uri = '//blp/apiauth';
+        var requestName = 'AuthorizationRequest';
+        this.doRequest(uri, requestName, request, callback, true, undefined);
```

Comment:
Will the `AuthorizationResponse` be partial response? If not, I guess we can simplify the code here too.